piptools/scripts/compile.py

```diff
@@ -45,7 +45,7 @@ def _get_default_option(option_name: str) -> Any:
 
 
 @click.command(context_settings={"help_option_names": ("-h", "--help")})
-@click.version_option()
+@click.version_option(package_name="pip-tools")
```

Comment:
@webknjaz should we rather not still stick to click >= 7 and pass this option only on click version 8?